text/2019-08-21-supported-platforms.md

```diff
@@ -0,0 +1,230 @@
+        - FreeBSD
+        - Any Linux using the 'Unbreakable Enterprise Kernel' by Oracle.
+* **Development Platforms**
+      + **Tier 1**:
+        - Any Linux kernel >3.x on X84-64 with glibc 2.17 with SSE4.2 or without
+        - The latest Mac OS X release on X86-64
+      + **Tier 2**:
+        - Any machine with access to a Docker installation which can support
+          Linux containers
+      + **Tier 3**:
+        - Nix (the package manager) environments
+        - Any Linux Kernel >3.x with musl libc on X86-64 or ARM64/AARCH64 with
+          SSE4.2 or without
+      + **Not supported**:
+        - Windows
+        - FreeBSD
+        - Any Linux using the 'Unbreakable Enterprise Kernel' by Oracle.
+
+This list does not attempt to list all posibilities, it lists all major targets
+we are asked about on a regular basis and special cases we are aware of. If
+something is not listed, it should be considered untested and in the "Not
+supported" tier.
+
+### RFC Effects
+
+If accepted, this RFC would:
+
+* **discourage** adding this data to the TiKV documentation.
+      - This should be a living development reference for our contributors and
+        community. We should not encourage new or end users to try unsupported
+        platforms, we'll limit our messaging to only Tier 1 platforms.
+      - These targets and how we support them may change in the future, and
+        maintaining separate lists is risks out of date data.
+* **encourage** stating and listing our tier 1 supported targets where
+  appropriate in documentation, as these specifics are of great importance to
+  our readers.
+* **encourage** any future changes be ammended to this RFC itself.
+
+This RFC as currently defined does not change any of our *current* support, it
+only clarifies them and defines a path forward for new targets.
+
+## Drawbacks
+
+* We must maintain this compatability specification and ensure correctness
+* Contributors may feel encouraged to try to promote certain platforms up a tier
+  and require us to do more testing on releases.
+* This list does not talk about hardware recommendations like CPUs and memory
+  since they are dependent on workload and too specific for this RFC. This is
+  about what hardware TiKV *can* and *can not* operate on, when not limited by
+  computational resources. Nor does it include specifics about which cloud, VM, or
+  container platforms we support. Users should refer to
+  https://tikv.org/docs/3.0/tasks/deploy/introduction/ for system specification
+  guidelines around these topics.
+
+## Alternatives
+
+* We could merge development and deployment tiers, but this would not be the
+  full story, since TiKV builds in more places than we support deployment.
+* We could not specify what we support and let our users enjoy a fun guessing game
+* We could only specify supported (Tier 1) targets and not worry about partial support.
+* We could limit our Tier 1 to platforms we run Jepsen/Schrodinger tests.
```

Comment:
Is this not the same as CentOS and Ubuntu?

Reply:
As far as I understand we only run Jepsen on CentOS right now.